image/prepare-base.sh

```diff
@@ -150,4 +150,11 @@ END
 sudo chmod +x /etc/init.d/gateway-iptables.sh
 sudo update-rc.d gateway-iptables.sh defaults
 
-echo "Done"
\ No newline at end of file
+# Check for an update every day
+sudo sh -c 'cat > /etc/cron.daily/mozilla-iot-gateway-update.sh' <<END
+#!/bin/bash
+su pi -c 'node /home/pi/mozilla-iot/gateway/tools/check-update.js'
+END
+sudo chmod +x /etc/cron.daily/mozilla-iot-gateway-update.sh
+
+echo "Done"
```

Comment:
Since this is running under the pi user, I think that we could launch this process from within the gateway itself. We just need to be careful to launch it as a detached process. This would allow the scheduling to be configured from within the gateway rather than relying on cron. Although I like the fact that cron will run regardless. Even if the gateway has badly crashed.

Reply:
I can switch the scheduling over to the gateway if you prefer.

If the gateway has crashed it should be picked up by the OnFailure handler which would then run `tools/rollback.sh`. The time when `cron` would be most useful is if `gateway_old` is also broken.

Reply:
Using cron is probably fine for now.